<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CryptocurrencyDialog</class>
 <widget class="QDialog" name="CryptocurrencyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>309</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="gbCryptocurrencyTransactionDetails">
     <property name="title">
      <string>Cryptocurrency Transaction Details</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblCryptocurrency">
        <property name="text">
         <string>Cryptocurrency :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cbCryptocurrency"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblOperationType">
        <property name="text">
         <string>Operation Type :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cbOperationType"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblPrice">
        <property name="text">
         <string>Price :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="dsbPrice">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblAmount">
        <property name="text">
         <string>Amount :</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="dsbAmount">
        <property name="decimals">
         <number>10</number>
        </property>
        <property name="maximum">
         <double>100000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblDateTime">
        <property name="text">
         <string>Date and Time :</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDateTimeEdit" name="dteDateTime">
        <property name="maximumDate">
         <date>
          <year>2250</year>
          <month>12</month>
          <day>31</day>
         </date>
        </property>
        <property name="minimumDate">
         <date>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
        <property name="displayFormat">
         <string>yyyy-MM-dd hh:mm:ss</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lblGoalPrice">
        <property name="text">
         <string>Goal Price :</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="dsbGoalPrice">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="bbTransactionApproval">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vlStatusBar"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bbTransactionApproval</sender>
   <signal>accepted()</signal>
   <receiver>CryptocurrencyDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bbTransactionApproval</sender>
   <signal>rejected()</signal>
   <receiver>CryptocurrencyDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
